pages/about.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="overflow-y-auto w-full">
```

Comment:
I think @theyokohamalife wants to keep this in

Reply:
Correct! This is to make sure the footer is stickied at the bottom of the page.

Reply:
To me this should stay as it creates two y-axis scrollbars and css should be used to make the footer sticky in the footer component.

On the production site it isn't sticky due to the main scrollbar existing.

![image](https://github.com/ourjapanlife/findadoc-web/assets/124335161/b8047e2f-9ae5-4e81-99cc-e3cb390f5363)

Reply:
@NabbeunNabi That is a bug caused by #564 and noted in issue  #596. Let's fix that first and exclude this for now. 🙏🏾